hl/src/H5LT.c

```diff
@@ -1898,7 +1898,6 @@ H5LTget_attribute_ndims(hid_t loc_id, const char *obj_name, const char *attr_nam
     /* End access to the attribute */
     if (H5Aclose(attr_id))
         goto out;
-    ;
```

Comment:
Weird that all these lone semicolons exist. I wonder where they all came from?

Reply:
I suspect someone fat-fingered `goto out;;` then clang-format put the second "statement" on a separate line.